lib/utilities/router.dart

```diff
@@ -2,12 +2,15 @@ import 'package:flutter/cupertino.dart';
 import 'package:flutter_ecommerce/utilities/routes.dart';
 import 'package:flutter_ecommerce/views/pages/bottom_navbar.dart';
 import 'package:flutter_ecommerce/views/pages/landing_page.dart';
-import 'package:flutter_ecommerce/views/pages/auth_page.dart';
+import 'package:flutter_ecommerce/views/pages/screens.dart';
 
 Route<dynamic> onGenerate(RouteSettings settings) {
   switch (settings.name) {
-    case AppRoutes.loginPageRoute:
+    case AppRoutes.navBottomContainerPageRoute:
```

Comment:
Why adding this route and there's one exists
You can refactor depend on it, just put the other things in separate widgets